portfolio/src/main/java/com/google/sps/data/Comment.java

```diff
@@ -1,6 +1,5 @@
 package com.google.sps.data;
 
-/** An item on a todo list. */
 public final class Comment {
```

Comment:
Public classes typically should have at least some javadoc. My recommendation is just a one liner.